<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Backup After</string>
     </property>
     <property name="buddy">
      <cstring>backupSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="backupSbx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="backupCmb"/>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QPushButton" name="backupBtn">
     <property name="text">
      <string>Backup Now</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="rollingBackupCbx">
     <property name="text">
      <string>Rolling Backup</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Rolling Backup Count</string>
     </property>
     <property name="buddy">
      <cstring>rollingCountSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="rollingCountSbx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="prefix">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="backupBeforeSaveCbx">
     <property name="text">
      <string>Backup Before Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Save Every</string>
     </property>
     <property name="buddy">
      <cstring>saveSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="saveSbx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="saveCmb"/>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>Save Now</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Skip First</string>
     </property>
     <property name="buddy">
      <cstring>skipSbx</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="skipSbx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Save Filename Prefix</string>
     </property>
     <property name="buddy">
      <cstring>savePrefixLed</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="savePrefixLed"/>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>55</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>backupSbx</tabstop>
  <tabstop>backupCmb</tabstop>
  <tabstop>backupBtn</tabstop>
  <tabstop>rollingBackupCbx</tabstop>
  <tabstop>rollingCountSbx</tabstop>
  <tabstop>backupBeforeSaveCbx</tabstop>
  <tabstop>saveSbx</tabstop>
  <tabstop>saveCmb</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>skipSbx</tabstop>
  <tabstop>savePrefixLed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
